config/examples/dashboards/policies.json

```diff
@@ -306,6 +306,148 @@
             "y": 8
          },
          "id": 9,
+         "title": "AuthPolicy",
+         "type": "row"
+      },
+      {
+         "datasource": {
+            "type": "prometheus",
+            "uid": "$datasource"
+         },
+         "description": "Total number of AuthPolicy across all clusters",
+         "gridPos": {
+            "h": 3,
+            "w": 2,
+            "x": 0,
+            "y": 9
+         },
+         "id": 10,
+         "pluginVersion": "v10.0.0",
+         "targets": [
+            {
+               "datasource": {
+                  "type": "prometheus",
+                  "uid": "$datasource"
+               },
+               "expr": "count(gatewayapi_authpolicy_status{name=~\"${authpolicy}\"})",
+               "instant": true
+            }
+         ],
+         "title": "Total",
+         "type": "stat"
+      },
+      {
+         "datasource": {
+            "type": "prometheus",
+            "uid": "$datasource"
+         },
+         "description": "Total AuthPolicy with an Available state",
+         "gridPos": {
+            "h": 3,
+            "w": 2,
+            "x": 2,
+            "y": 9
+         },
+         "id": 11,
+         "pluginVersion": "v10.0.0",
+         "targets": [
+            {
+               "datasource": {
+                  "type": "prometheus",
+                  "uid": "$datasource"
+               },
+               "expr": "count(gatewayapi_authpolicy_status{type=\"Available\", name=~\"${authpolicy}\"})",
```

Comment:
I guess this would allow us to define as well a PrometheusRule alert for when the policy is not available?

(Soon to evolve to proper GWAPI-compliant status conditions: https://docs.kuadrant.io/architecture/rfcs/0004-rlp-status/#conditions.)

Reply:
Yes, exactly.
This metric enables both alerting rule capabilities and historical status tracking for troubleshooting. 
There's an issue semi related to this for showing status on a graph if you're interested in chiming in #16 .
There's also a set of example alert rules in https://github.com/Kuadrant/gateway-api-state-metrics/blob/main/config/examples/rules/alert-rules.yaml that could be extended.